challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,16 @@
+function binarySearch(array, searchKey) {
```

Comment:
Not really a binary search, but the directions never really specified. :)

Reply:
Ah! I did not understand that.
I rewrote it so it's now a binary search

Reply:
nice job!